src/utils/sources-tree/getURL.js

```diff
@@ -37,7 +38,8 @@ function _getURL(source: Source, defaultDomain: string): ParsedURL {
     return def;
   }
 
-  const { pathname, protocol, host } = parse(url);
+  const { pathname, searchParams, protocol, host } = parse(url);
```

Comment:
It would be better to use the `search` property instead.  The value of `searchParams` is an object, and we don't need an object; `search` provides just the string `s=1&t=2`, etc.

Reply:
I have replaced `searchParams` with `search` on my computer. Should I push to the origin and make a new pull request? Can I do the changes in this pull request?